tiledb/multirange_indexing.py

```diff
@@ -32,10 +32,17 @@
 except ImportError:
     DataFrame = None
 
-
-# sentinel value to denote selecting an empty range
 EmptyRange = object()
 
+
+def is_empty_range(idx: Any) -> bool:
+    if idx is not EmptyRange:
+        if hasattr(idx, "__len__") and len(idx) == 0 and idx != "":
```

Comment:
Please double-check numpy semantics here -- is it any empty collection, or only `list`?

Reply:
`Union[EmptyRange, List, Tuple]` works fine after changing type of `EmptyRange` from `object` to `TypeVar`.